lang/en/texts/android.html

```diff
@@ -0,0 +1,30 @@
+<h4>Mobile</h4>
+<a download="4. Additives - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png"></a>
+<a download="8. Lists - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/8. Lists - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/8. Lists - Mobile - EN.png"></a>
+<a download="3. Compare - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/3. Compare - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/3. Compare - Mobile - EN.png"></a>
+<a download="9. Mission - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/9. Mission - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/9. Mission - Mobile - EN.png"></a>
+<a download="7. Allergen alert - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/7. Allergen alert - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/7. Allergen alert - Mobile - EN.png"></a>
+<a download="2. Good for you - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/2. Good for you - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/2. Good for you - Mobile - EN.png"></a>
+<a download="6. History - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/6. History - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/6. History - Mobile - EN.png"></a>
+<a download="5. Search - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/5. Search - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/5. Search - Mobile - EN.png"></a>
+<a download="1. Scanner - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/1. Scanner - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/1. Scanner - Mobile - EN.png"></a>
+<h4>Square</h4>
```

Comment:
I'm not quite sure what `square` is supposed to mean in this context.

Reply:
Those are Instagram / Social media images that are square.